tests/pipelines/test_pipelines_question_answering.py

```diff
@@ -106,11 +106,13 @@ def run_pipeline_test(self, question_answerer, _):
         self.assertEqual(outputs, {"answer": ANY(str), "start": ANY(int), "end": ANY(int), "score": ANY(float)})
 
         # Using batch is OK
+        if question_answerer.tokenizer.pad_token_id is None:
+            question_answerer.tokenizer.pad_token_id = question_answerer.model.config.eos_token_id
```

Comment:
Since batching can now be used, we need to actually enable batching even when tokenizer don't have a specific padding token.